dell-csi-helm-installer/csi-install.sh

```diff
@@ -147,6 +145,8 @@ function install_driver() {
   HELMOUTPUT="/tmp/csi-install.$$.out"
   run_command helm ${1} \
     --set openshift=${OPENSHIFT} \
+    --values "${DRIVERDIR}/${DRIVER}/k8s-${kMajorVersion}.${kMinorVersion}-values.yaml" \
+    --values "${DRIVERDIR}/${DRIVER}/driver-image.yaml" \
```

Comment:
I'm confused on this part. 
These two yamls are no longer present in this repo, are people supposed to create them?